web/beacon-app/src/features/projects/components/RenameProjectModal.tsx

```diff
@@ -0,0 +1,51 @@
+import { Modal } from '@rotational/beacon-core';
+import { useEffect } from 'react';
+import { toast } from 'react-hot-toast';
+
+import { useUpdateProject } from '../hooks/useUpdateProject';
+import { Project } from '../types/Project';
+import RenameProjectModalForm from './RenameProjectModalForm';
+
+type ChangeRoleModalProps = {
+  open: boolean;
+  project: Project | null;
+  handleModalClose: () => void;
+};
+
+// eslint-disable-next-line unused-imports/no-unused-vars
+function RenameProjectModal({ open, handleModalClose, project }: ChangeRoleModalProps) {
+  const { updateProject, wasProjectCreated } = useUpdateProject();
+
+  useEffect(() => {
+    if (wasProjectCreated) {
+      toast.success('Success! You have renamed your project.');
+    }
+  }, [wasProjectCreated]);
+
+  const handleSubmit = (values: any) => {
+    const payload = {
+      projectID: project?.id || '',
+      projectPayload: {
+        name: values['new-name'],
+      },
+    };
+    updateProject(payload);
+    handleModalClose();
+  };
+
+  return (
+    <Modal
+      open={open}
+      title="Rename Project"
+      containerClassName="overflow-scroll  max-w-[80vw] lg:max-w-[50vw] no-scrollbar"
+      data-testid="keyCreated"
```

Comment:
I believe this test ID is used for the modal that appears when an API key modal is created, so this should likely be updated to agree better with what is happening here.

Reply:
just forgot to change it since it was taken from another modal component.